src/main/java/com/smartbear/mqttsupport/connection/dialog/EditConnectionDialog.java

```diff
@@ -643,9 +646,9 @@ protected boolean handleOk() {
             }
         }
 
-        if (!checkCertificateField(caCertificateEdit)
-                || !checkCertificateField(clientCertificateEdit)
-                || !checkCertificateField(privateKeyEdit)) {
+        if (!checkCertificateField(caCertificateEdit, expandProperty(caCertificateEdit.getText()))
```

Comment:
Usage of expandProperty method should be nested in checkCertificateField method instead of changing it's signature.